tcf_website/templates/reviews/new_review.html

```diff
@@ -48,13 +51,28 @@ <h1 class="h2 mt-2 mb-4">Write a review</h1>
                 if(check.duplicate) {
                     $("#duplicate-review").modal();
                 } else {
-                    // Timeout button for 3 seconds so user can't spam button while form is submitting
-                    $("#submitBtn").prop("disabled", true);
-                    setTimeout(enableButton, 3000);
-                    // $("#reviewForm") didn't work for some reason
-                    // Not a duplicate review, so proceed with normal form submission for new review
-                    document.getElementById("reviewForm").action = "{% url 'new_review' %}";
-                    document.getElementById("reviewForm").submit();
+                    // Check if review has 0 total hours/week and warn user if it does
+                    $.ajax({
+                        type: "POST",
+                        url: "/reviews/check_zero_hours_per_week",
+                        data: $("#reviewForm").serialize(),
+                        async: false,
+                        success: function (check) {
+                            // Only show "zero-hours-review" modal once
+                            if(check.zero && count == 0) {
+                                $("#zero-hours-review").modal();
+                                count = 1;
+                            } else {
+                                // Timeout button for 3 seconds so user can't spam button while form is submitting
+                                $("#submitBtn").prop("disabled", true);
+                                setTimeout(enableButton, 3000);
+                                // $("#reviewForm") didn't work for some reason
+                                // Not a duplicate review, so proceed with normal form submission for new review
```

Comment:
Comment should be updated to reflect how at this point not only is the review not a duplicate, but that the user has been shown 0 hours modal

Reply:
^ has been shown it already or it was never necessary to show in the first place